bench/playbooks/roles/wkhtmltopdf/tasks/main.yml

```diff
@@ -20,11 +20,17 @@
     force: yes
   when: ansible_os_family == 'Debian'
 
-- name: download wkthmltox Ubuntu 20
+- name: download wkthmltox Ubuntu 20 amd64
```

Comment:
This change is unnecessary IMO since this will still be the most widely used title

Reply:
I disagree, since this part of code is downloading arch-specific files (instead of say, `pip install` or `apt-get` where you kind of not care what's the underlying arch).

But I have applied your suggestion.

Reply:
I'm not entirely clear about this. But looks fine as is now.

Reply:
I've removed the "amd64" in latest update